hudi-common/src/main/java/org/apache/hudi/common/table/log/AbstractHoodieLogRecordReader.java

```diff
@@ -151,6 +155,14 @@ protected AbstractHoodieLogRecordReader(FileSystem fs, String basePath, List<Str
     this.instantRange = instantRange;
     this.withOperationField = withOperationField;
     this.enableFullScan = enableFullScan;
+
+    // virtual keys handling
+    if (!tableConfig.populateMetaFields()) {
+      this.virtualKeysEnabled = true;
+      this.simpleKeyGenFields = Option.of(
+          Pair.of(tableConfig.getRecordKeyFieldProp(), tableConfig.getPartitionFieldProp()));
+    }
+    this.partitionName = partitionName;
```

Comment:
partitionPath to be consistent with everything else

Reply:
Path refers to the full path. Name is the relative part. I think we need the latter here so its not a path actually.

Reply:
Right, we want partition name here.